electron_app/package.json

```diff
@@ -5,11 +5,16 @@
   "version": "1.4.2",
   "description": "A feature-rich client for Matrix.org",
   "author": "New Vector Ltd.",
+  "scripts": {
+    "build": "electron-build-env --electron 6.0.3 neon build seshat-node --release",
+    "postinstall": "yarn build"
+  },
   "dependencies": {
     "auto-launch": "^5.0.1",
     "electron-store": "^2.0.0",
     "electron-window-state": "^4.1.0",
     "minimist": "^1.2.0",
-    "png-to-ico": "^1.0.2"
+    "png-to-ico": "^1.0.2",
+    "matrix-seshat": "^0.3.0"
```

Comment:
I assume there will more be changes to move this to documentation in some way?

Reply:
Yeah, I wanna leave it in until it's ready to be merged in case a curious soul wants to try the PR out.